csrc/python_frontend/fusion_definition.h

```diff
@@ -240,6 +250,17 @@ class NVF_API FusionDefinition : public FusionState {
 
  private:
   mutable std::optional<std::string> debug_output_ = std::nullopt;
+
+#ifdef NVFUSER_DISTRIBUTED
+  //! DO NOT COMMIT TO THESE CHANGES!
+  //! The reason we have these is due to the lack of cache for multidevice
+  //! executor
+  bool multidevice_flag = false;
```

Comment:
```suggestion
  bool multidevice_flag_ = false;
```

Reply:
There's also `multidevice_*` vs `multi_device_*`. I slightly prefer the former because multi-device (despite of the hypen) is an English word. But we should at least make the names in this file consistent.